scripts/UpdateDependencies.ps1

```diff
@@ -55,6 +55,8 @@ foreach ($package in $remoteDeps.SelectNodes('//Package')) {
     }
 }
 
+$globalObj = Get-Content "$PSScriptRoot/../global.json" -Raw | ConvertFrom-Json
+$variables["SdkPackageVersion"] = @($globalObj.sdk.version)
```

Comment:
This isn't actually updating to a new version. To make this work, we'll need to figure out which ProdCon version variable to use, and then update the global.json file.